### DISCUSSION TITLE
Duplicate MRs created after fixing `cacheDir` setting

### DISCUSSION THREAD

**Discussion by rassie** [renovatebot/renovate#15282] [Request Help]
I'm running a self-hosted Renovatebot in a CronJob in OpenShift. Today I've found out that my configuration was wrong and `cacheDir` hasn't been pointing to a volume as originally intended, i.e. there has been effectively no cache for any Renovatebot run. After I've fixed that and ran the bot, every single open MR has been opened again, i.e. duplicated. As far as I can see, Renovate does not recognize the old ones anymore, since they are not referenced in the dashboard issue.

What can I do about it? Can I make Renovatebot recognize old MRs again and close the new ones (or vice versa)?

**Comment by rarkins** [comment]
Renovate uses these three things to match old (I assume you mean closed/ignored?) PRs:
- user account which created them
- branch name
- PR title

Can you work out which of those three changed?

**Reply by rassie** [reply]
Thanks, it was the branch name.

I don't know whether this was an error of mine from the beginning or a change in Renovate Bot over last couple of years, but several environment variables I used had wrong names, e.g. `RENOVATE_ONBOARDINGCONFIG` instead of `RENOVATE_ONBOARDING_CONFIG` and also `RENOVATE_BRANCHPREFIX` instead of `RENOVATE_BRANCH_PREFIX`. I've updated them more or less mindlessly and missed that my branch prefix has now been set to `renovate-` instead of the default (and used before) `renovate/`.  I've removed the setting altogether.

What would be the most efficient way to remove the wrong MRs? Would it work if I just deleted the wrongly named branches or do I need to close the MRs?

**Reply by rarkins** [reply]
I'm not sure for gitlab. With GitHub it will close the associated PR when the branch is deleted

**Reply by rassie** [reply]
I've ended up deleting MRs per script, will need to cleanup branches someday. Thanks again!
